config/locales/views.fr.yml

```
@@ -435,9 +435,9 @@ fr:
       thats_you: C’est vous !
   needs:
     archive:
-      archive: Mettre en échec
-      subjet_achived: Le sujet a été mis en échec
-      unarchive: Annuler la mise en échec
+      archive: Archiver
+      subjet_achived: Le besoin a été archivé
+      unarchive: Annuler l'archivage
```

```suggestion
      unarchive: Désarchiver
```
[c’est même un mot dans le dictionnaire :grin: ](https://fr.wiktionary.org/wiki/désarchiver)